boards/rcar_h3ulcb_ca57.overlay

```diff
@@ -0,0 +1,605 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/delete-node/ &sd0;
+/delete-node/ &vcc_sd0;
+/delete-node/ &vccq_sd0;
+/delete-node/ &gpio5;
+
+/ {
+	domu_configs {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		/* it is needed only for correct filling structures by macros */
```

Comment:
Looks like hack. Is there a way to get rid of this?

Reply:
It is the simplest way that I found to do it. It is possible to convert an array into an array of structures, but we would need to do it at runtime, or it would require a lot of changes for macro processing (like multilevel COND_ with checking array indexes).